app/com/m3/octoparts/aggregator/service/PartResponseLocalContentSupport.scala

```diff
@@ -0,0 +1,29 @@
+package com.m3.octoparts.aggregator.service
+
+import com.m3.octoparts.aggregator.PartRequestInfo
+import com.m3.octoparts.model.PartResponse
+import com.m3.octoparts.model.config.{ HttpPartConfig, ShortPartParam }
+
+import scala.concurrent.Future
+
+trait PartResponseLocalContentSupport extends PartRequestServiceBase {
+
+  override def processWithConfig(ci: HttpPartConfig,
+                                 partRequestInfo: PartRequestInfo,
+                                 params: Map[ShortPartParam, Seq[String]]): Future[PartResponse] = {
+    if (ci.localContentsEnabled) {
```

Comment:
@mauhiz I believe this feature is to prevent the registered target backend from getting hit **period**. Apparently, *certain backends* have times when they cannot sustain any more traffic and might need some kind of ... escape hatch.